tests/pytest/test_multi.py

```diff
@@ -66,7 +66,7 @@ def testDelCommand(env):
     r.assertEqual(res, 3)
 
     r.assertOk(r.execute_command('JSON.SET', 'doc2', '$', '[1, 2, 3]'))
-    res = r.execute_command('JSON.DEL', 'doc2', '$[2,1,0]')
+    res = r.execute_command('JSON.DEL', 'doc2', '$[1,2,0]')
```

Comment:
Why this change?

Reply:
It was initially in a none-descending order, but was worked around a bug, and now can be restored to a less trivial order which would test the reordering required by DEL.

Reply:
So let's keep them both?